demo-app/src/pages/default/index.jsx

```diff
@@ -9,6 +9,18 @@ import {
   ReverseAnchor,
 } from '../../components';
 
+const tileLayer = {
+  // url: 'https://tiles.stadiamaps.com/tiles/alidade_smooth/{z}/{x}/{y}{r}.png',
```

Comment:
Remove comment

Reply:
modified comments to communicate intent (e.g. PMTiles won't work until we get a PMTiles file on CDN)